src/main/java/com/alkemy/ong/entity/ContactEntity.java

```diff
@@ -27,7 +27,7 @@
     @NotEmpty(message = "The field must not be empty.")
     private String name;
     private String phone;
-    @Column(name = "email", nullable = false, unique = true)
+    @Column(name = "email", nullable = false, unique = false)
```

Comment:
why is the mail not unique? can two contacts have the same address?

Reply:
It doesn't have to be that way, because later the Frontend developer would handle it with an asynchronous process.

Reply:
I understand, thank you!